### DISCUSSION TITLE
'conda' is not recognized as an internal or external command, operable program or batch file.

### DISCUSSION THREAD

**Discussion by AIZeplin** [oobabooga/text-generation-webui#1492] [Q&A]
Hi! Whenever I try to run the command line to start it up, I get the error "'conda' is not recognized as an internal or external command,
operable program or batch file." I've tried to go ahead and install Miniconda on my computer (Windows11), but it did nothing and still get the same error.
Can anyone help me on what I'm doing wrong? What else I need to install?

**Comment by AIZeplin** [comment]
ChatGPT eventually gave me the correct answer. Conda was installed, but no path was added. If you have the same problem, this is how you fix it:

To fix this issue, you can try the following steps:

Find the installation directory of Conda. The default installation directory of Conda is typically C:\Users\your-username\miniconda3 on Windows. If you installed Conda to a different directory, you will need to adjust the following steps accordingly.

Open the Start menu and search for "Environment Variables" or "Edit the system environment variables". Click on the "Edit the system environment variables" option.

In the System Properties window, click on the "Environment Variables" button.

In the "Environment Variables" window, under the "System variables" section, scroll down and find the "Path" variable. Click on the "Edit" button.

In the "Edit environment variable" window, click on the "New" button.

Enter the full path to the "Scripts" and "condabin" directories of your Conda installation. For example, if your Conda installation directory is C:\Users\your-username\miniconda3, then you should add the following paths:

makefile
Copy code
C:\Users\your-username\miniconda3\Scripts
C:\Users\your-username\miniconda3\condabin
Click on "OK" to close all windows.

Open a new command prompt window and try running the conda command.

After following these steps, you should be able to use the conda command in your terminal.

**Comment by AIZeplin** [comment]
Sadly, this resulted in the next line of errors/problem:

C:\AI\oobabooga_windows>conda activate textgen

CommandNotFoundError: Your shell has not been properly configured to use 'conda activate'.
If using 'conda activate' from a batch script, change your
invocation to 'CALL conda.bat activate'.

To initialize your shell, run

    $ conda init <SHELL_NAME>

Currently supported shells are:
  - bash
  - cmd.exe
  - fish
  - tcsh
  - xonsh
  - zsh
  - powershell

See 'conda init --help' for more information and options.

IMPORTANT: You may need to close and restart your shell after running 'conda init'.

**Reply by AIZeplin** [reply]
I eventually fixed this by using a different set of commands to start the script, specifically:
CALL conda.bat activate textgen

**Comment by ZacharyDK** [comment]
Have the same problem, but on mac....
